app/templates/main/upload.html

```diff
@@ -15,68 +15,66 @@
-                    {% if days_to_deadline < 0 %}
-                        {% set notificationClass %} {{ "overdue-notification-banner" }} {% endset %}
-                    {% endif %}
+            {{ govukNotificationBanner ({
+                "html": notificationHTML,
+                "classes": notificationClass
+            }) }}
+        {% endif %}
 
-                    {{ govukNotificationBanner ({
-                        "html": notificationHTML,
-                        "classes": notificationClass
-                    }) }}
-                {% endif %}
+        {{ uploadTable(local_authorities, fund, reporting_period) }}
 
-                {{ uploadTable(local_authorities, fund, reporting_period) }}
-
-                <div class="upload-data-container govuk-!-margin-top-5">
-                    <form method="post" action="{{ url_for('main.upload') }}" enctype="multipart/form-data">
-                        <h2 class="govuk-heading-m">Upload your data return</h2>
-                        {% if pre_error %}
-                            {{ govukFileUpload({'id': 'ingest_spreadsheet',
-                             "hint": {"text": ""},
-                             "label": {"text": ""},
-                             "errorMessage": {"html": errorMessage(pre_error)},
-                             'name': 'ingest_spreadsheet'}) }}
-                        {% else %}
-                            {{ govukFileUpload({'id': 'ingest_spreadsheet',
-                             "hint": {"text": ""},
-                             "label": {"text": ""},
-                             'name': 'ingest_spreadsheet'}) }}
-                        {% endif %}
-                        <p class="govuk-body">When you upload your return, we’ll check it for missing data and formatting errors.</p>
+        <div class="upload-data-container govuk-!-margin-top-5">
+            <form method="post" action="{{ url_for('main.upload') }}" enctype="multipart/form-data">
+                <h2 class="govuk-heading-m">Upload your data return</h2>
+                {% if pre_error %}
+                    {{ govukFileUpload({'id': 'ingest_spreadsheet',
+                     "hint": {"text": ""},
+                     "label": {"text": ""},
+                     "errorMessage": {"html": errorMessage(pre_error)},
+                     'name': 'ingest_spreadsheet'}) }}
+                {% else %}
+                    {{ govukFileUpload({'id': 'ingest_spreadsheet',
+                     "hint": {"text": ""},
+                     "label": {"text": ""},
+                     'name': 'ingest_spreadsheet'}) }}
+                {% endif %}
+                <p class="govuk-body">When you upload your return, we’ll check it for missing data and formatting errors.</p>
 
-                    {{ govukButton({
-                        "element": "button",
-                        "text": "Upload and check for errors"})
-                    }}
+            {{ govukButton({
```

Comment:
I indented this button in my previouse PR to be at the same level as the <p> above it. But it doesn't matter either way

Reply:
Good spot - I've re-indented it. 

I wonder if there is a way we can all install a pre-commit to make this consistent... I did have a brief look at one point but didn't have much luck

Reply:
like black for python code